<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulation_input</class>
 <widget class="QDialog" name="simulation_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Carcamo Level</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Reactor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Motor 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Motor 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Caudal in</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="1">
    <widget class="QScrollBar" name="horizontalScrollBar_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="8" column="1">
    <widget class="QScrollBar" name="horizontalScrollBar_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Caudal out</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="10" column="1">
    <widget class="QScrollBar" name="horizontalScrollBar_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
